app/controllers/api/v1/users_controller.rb

```diff
@@ -4,7 +4,7 @@ class API::V1::UsersController < API::V1::APIController
   ## TODO remove support for leads overriding generated passwords once the site 
   ## is opened up to anyone.
   def create
-    existing_user = User.find_by(email: user_params[:email])
+    existing_user = User.find_by(email: user_params[:email].downcase)
```

Comment:
@littleforest not urgent but in an upcoming release, can we slip in something that'll ensure we're downcasing every email that comes in for user? Bake it right into the `email=` setter so we only ever have downcased emails to deal with in the DB?

Reply:
Sure will do